docs/source/releasenotes/3.1.rst

```diff
@@ -31,3 +31,10 @@ Bug Fixes
 ^^^^^
 
 - [KC-5909] - Synchronizing index entry and record writes
+
+3.1.3
+^^^^^
+
+- [KC-5962] - Ability to disable term type detection on PySpark Transform output
```

Comment:
Actually, term type detection was removed so this should say something like "Term type detection removed on PySpark transform output".

Reply:
fixed